ush/run_fv3.ksh

```diff
@@ -0,0 +1,40 @@
+#!/bin/ksh --login
```

Comment:
Can this be a bash script to match all the others in the regional_workflow, please? I don't think anything needs to change below since it's pretty straightforward. Just the name of the file and this line.

Reply:
Renamed the script name and changed that head line. Thank you for the suggestion!